docs/source/modules/installation/installation.rst

```diff
@@ -175,11 +175,20 @@ Some aliases can be defined for convenience.
   alias python3="docker exec my-merlin python3"
 
 When you are done with the containers you can stop them using ``docker-compose down``.
+We will be using the containers in the subsequent modules so leave them running.
```

Comment:
this file change seems like it's from a different PR?

Reply:
I added tutorial doc changes since tutorial was moved into develop.